packages/hash/api/src/lib/schemas/jsonSchema.ts

```diff
@@ -175,13 +175,14 @@ export class JsonSchemaCompiler {
    * @param description optional description for the type
    * @returns schema the complete JSON schema object
    */
-  async jsonSchema(
-    title: string,
-    accountId: string,
-    entityTypeId: string,
-    maybeStringifiedSchema: string | JSONObject = {},
-    description?: string,
-  ) {
+  async jsonSchema(params: {
+    $id?: string | null;
```

Comment:
Why accept `null` as a value for `$id`?
```suggestion
    $id?: string;
```
(This would also save you from having to do `$id: $id ?? undefined` later on)

Reply:
Good point, it isn't a function param, so it can just be left out instead of set to null :+1: